src/scenes/AcceleratorRouter/Cohorts/CohortsTable.tsx

```diff
@@ -43,6 +43,11 @@ const columns = (activeLocale: string | null): TableColumnType[] =>
           name: strings.PHASE,
           type: 'string',
         },
+        {
+          key: 'participantIds',
```

Comment:
Looks like the BE search will return a `numParticipants` count. You can use that, might simplify the rendering too.

Reply:
Didn't realize that, thanks for mentioning. Will update.

Reply:
I just checked the `cohorts` data with the `depth` param set to `'Participant'` for `requestCohorts` and the `numParticipants` property doesn't seem to be available. Did a quick search and wasn't able to find it in the API schema either.

Reply:
It is in an upcoming PR. It will be part of the search response, see `SearchResponsePayload` in the generated-types (it is very generic).
We can make the change later for sure.